### ISSUE TITLE
Codiad Live Demo Dead

### ISSUE THREAD

**Issue by innXctive** [Codiad/Codiad#1137]
### Hey mates!

Just to inform you [http://demo.codiad.com/](http://demo.codiad.com/) URL **is not working**:

`ERR_CONNECTION_RESET`

Anyway, I find your project genuinely **AWESOME**

Hope you'll deal with that.

Bye

### P.S.:

Here is my UA, dunno if it's helpfu...

`[Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.41 Safari/537.36 Edg/101.0.1210.32](https://developers.whatismybrowser.com/useragents/parse/?analyse-my-user-agent=yes)`

**Comment by HLSiira** [comment]
Hey, Codiad hasn't seen much updating in several years, but I've been supporting an active fork of it called Atheos for a long while now, if you're interested in it. It's almost a complete rewrite, with more built in features.

**Comment by innXctive** [comment]
> Hey, Codiad hasn't seen much updating in several years, but I've been supporting an active fork of it called Atheos for a long while now, if you're interested in it. It's almost a complete rewrite, with more built in features.

Thanks, such a quick answer 😅 Pls send me the link to your fork

**Comment by HLSiira** [comment]
Yeah no problem, I was playing Wordle on my phone when the notification popped up.

Main site is https://www.atheos.io
Repo is https://github.com/Atheos/Atheos

Let me know what you think!
